mobile/lib/shared/models/asset.dart

```diff
@@ -185,6 +190,12 @@ class Asset {
   @ignore
   bool get isImage => type == AssetType.image;
 
+  @enumerated
+  BackupSelection selectedForBackup;
+
+  @Backlink(to: Album.assetsLinkId)
+  final IsarLinks<LocalAlbum> localAlbums = IsarLinks<LocalAlbum>();
```

Comment:
I'd like to avoid this as well as we can't move assets across isolates when they contain links